### PR TITLE
#36 Don't generate setter for final @EpoxyAttribute fields

### PR DESCRIPTION
**Pull request by geralt-encore**
https://github.com/airbnb/epoxy/issues/36

### CODE REVIEW

File: epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java [hunk 1]
```diff
@@ -160,7 +161,7 @@ private void validateAccessibleViaGeneratedCode(Element attribute) throws
 
     // Verify method modifiers.
     Set<Modifier> modifiers = attribute.getModifiers();
-    if (modifiers.contains(PRIVATE) || modifiers.contains(STATIC) || modifiers.contains(FINAL)) {
+    if (modifiers.contains(PRIVATE) || modifiers.contains(STATIC)) {
       throwError(
           "%s annotations must not be on private, final, or static fields. (class: %s, field: %s)",
```
Comment (elihart) [inline epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java]: update the error message to not include final

Comment (geralt-encore) [inline epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java]: done

**Comment by elihart** [comment]
looks good to me, but not sure why the tests are failing. if you can fix them I'll merge, otherwise when I have time later I can pull the branch and figure out why the tests are failing

**Comment by geralt-encore** [comment]
`testModelWithFinalClassFails` was wrong from the start, because in contained static field annotated with `@EpoxyAttribute` and failing because of it, but not because the class was actually final.

**Comment by elihart** [comment]
Ah, my bad. Thanks for finding the issue and fixing! Looks great.

**Pull request merged**